doc/classes/ProjectSettings.xml

```diff
@@ -353,6 +353,10 @@
 			This setting can be overridden using the [code]--max-fps &lt;fps&gt;[/code] command line argument (including with a value of [code]0[/code] for unlimited framerate).
 			[b]Note:[/b] This property is only read when the project starts. To change the rendering FPS cap at runtime, set [member Engine.max_fps] instead.
 		</member>
+		<member name="application/run/enable_alt_space_menu" type="bool" setter="" getter="" default="false">
```

Comment:
Needs to be in alphabetical order, so this should be before `flush_stdout_on_print`